### ISSUE TITLE
Splitting the same file dims both splits

### ISSUE THREAD

**Issue by atav32** [blueyed/vim-diminactive#20]
1. open a file
2. type `:split`
3. both windows dim
4. type `:filetype detect` (mapped to `:Re` in my vimrc) and colorscheme comes back for *both splits*
5. switch windows, both windows dim

![sfal6ndxtf](https://cloud.githubusercontent.com/assets/622185/13299308/6b803262-daef-11e5-87c9-05fe6b163ecf.gif)

in my .vimrc
```
let g:diminactive_use_colorcolumn = 0
let g:diminactive_use_syntax = 1
```

**Comment by atav32** [comment]
oops, just saw this https://github.com/blueyed/vim-diminactive/issues/10

**Comment by blueyed** [comment]
Can you think of something to improve this situation?

**Comment by blueyed** [comment]
This is fixed by now via #10.
